src/tsconfig.rs

```diff
@@ -18,16 +14,16 @@ pub struct TsConfig {
     #[serde(skip)]
     path: PathBuf,
 
-    /// The value of extends must be a string containing a path to another configuration file to inherit from.
-    /// https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#configuration-inheritance
-    #[serde(default)]
-    pub extends: Option<String>,
+    // Base url for compiler option paths.
+    #[serde(skip)]
+    paths_base: PathBuf,
```

Comment:
Moved this up from the inner compiler options, since this field doesn't exist there.